feat: additonal mmsu start playbook for upgrade scenario (VIYAARK-336) (#146)
sassoftware/viya-ark: playbooks/viya-mmsu/README.md

```
ansible-playbook viya-ark/playbooks/viya-mmsu/viya-services-enable.yml
```

> To start the minimally required set of services prior to starting a Hotfix update, execute:

Reviewer: we should remove the word Hotfix, it's rarely used in documentation, and the doc refers to it just as an "update" process
https://go.documentation.sas.com/doc/en/calcdc/3.5/dplyml0phy0lax/n0f5bkzrsoh6bnn1jc3x1n1b5lye.htm#n11ptxz3swm48pn117wqpa2m7g2a

> An update replaces some or all of your deployed software with the latest instances within the product version of the software. You perform the update with the same software order that was used to install SAS Viya.

```suggestion
To start the minimally required set of services prior to updating your software, execute:
```
